src/derivative_wrappers.jl

```diff
@@ -574,12 +572,12 @@ function _jacNoise!(λ, y, p, t, S::TS, isnoise::Bool, dgrad, dλ, dy) where TS<
 
     if StochasticDiffEq.is_diagonal_noise(prob)
       pJt = transpose(λ).*transpose(pJ)
-      dgrad[:] .= vec(pJt)
+      dgrad[:] .= -vec(pJt)
```

Comment:
Instead of these changes, we could also have a 

```julia
dgrad .*= -1
```
similar to the 
```julia
dλ .*= -1
```
within `BacksolveAdjoint` and `InterpolatingAdjoint`. Might actually be easier to read along the typical definition given in https://github.com/SciML/DiffEqSensitivity.jl/pull/545#issue-1105053984.

Reply:
`-vec` allocates, so we shouldn't do that.

Reply:
oh yes. Then the other option is definitely better. (I guess `.-vec` could be fine but that's definitely harder to read I think)